torch/csrc/jit/passes/graph_fuser.cpp

```diff
@@ -127,22 +141,24 @@ struct GraphFuser {
       return false;
     }
   }
-  bool allFloatIO(Node * node) {
-    for(auto & o : node->outputs()) {
-      if(!hasFloatType(o)) {
-        return false;
-      }
-    }
-    for(auto & o : node->inputs()) {
+  bool allFloatList(at::ArrayRef<Value*> list){
+    for (auto & o: list){
       if(!hasFloatType(o)) {
         return false;
       }
     }
     return true;
   }
+  bool allFloatIO(Node * node) {
+    if (!allFloatList(node->outputs())) return false;
+    if (!allFloatList(node->inputs())) return false;
+    return true;
```

Comment:
This is completely equivalent to `allFloatList(node->inputs()) && allFloatList(node->outputs())`. Let's use that, because it's much simpler.